pkg/podrescli/client.go

```diff
@@ -3,13 +3,13 @@ package podrescli
 import (
 	"context"
 	"fmt"
-	"log"
 	"os"
 	"reflect"
 	"strings"
 	"time"
 
 	"google.golang.org/grpc"
+	"k8s.io/klog/v2"
```

Comment:
This line should be together with the other k8s dependencies

Reply:
right, fixed.